server/routes/prisoner/handlers/overview.ts

```diff
@@ -44,6 +44,10 @@ export default class OverviewRoutes {
           ? await this.calculateReleaseDatesService.hasIndeterminateSentences(bookingId, token)
           : false
 
+      const requiresNewCalculation = config.featureFlags.thingsToDo
```

Comment:
this could be added to the `Promise.all` earlier on, so its not blocking here.

Reply:
Have left this one as-is for now with a TODO to move it when removing the feature flag, so we don't make unneeded calls if flagged off